From 32d81a88ad076538d4d1e4ac88dda01fc2165b30 Mon Sep 17 00:00:00 2001
From: Samuel Dionne-Riel <samuel@dionne-riel.com>
Date: Sat, 28 Mar 2020 01:33:16 -0400
Subject: [PATCH] dts: pinephone: Setup default on and panic LEDs

 * The green LED defaults to on.
 * The red LED is the panic indicator.

The green LED defaults to on for an expected transition through red,
yellow and green during the different boot stages.
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
index 93bc3238bf3d..e67510e0e505 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
@@ -63,6 +63,7 @@ green {
 			color = <LED_COLOR_ID_GREEN>;
 			gpios = <&pio 3 18 GPIO_ACTIVE_HIGH>; /* PD18 */
 			retain-state-suspended;
+			linux,default-trigger = "default-on";
 		};
 
 		red {
@@ -70,6 +71,7 @@ red {
 			color = <LED_COLOR_ID_RED>;
 			gpios = <&pio 3 19 GPIO_ACTIVE_HIGH>; /* PD19 */
 			retain-state-suspended;
+			panic-indicator;
 		};
 	};

